docs/md/staged-executor.md

```diff
@@ -51,7 +49,6 @@ The table below summarises the purpose of each attribute in the above request.
 | `name`                       | Selects the `Terra` executor implementation.                                                      |
 | `workspace`                  | Terra Workspace in which to execute workflows.                                                    |
 | `methodConfiguration`        | Method configuration from which to generate submissions.                                          |
-| `methodConfigurationVersion` | Expected version of the method configuration.                                                     |
 | `fromSource`                 | Instruction to coerce an output from an upstream `Source` to a type understood by this `executor`.|
```

Comment:
… and … again … just formatting, I know …

Reply:
All good, appreciate the eyes.